.github/workflows/e2e-vm.yml

```
@@ -0,0 +1,208 @@
+  PVE_AUTH: "PVEAPIToken=${{ secrets.CI_RUNNER_API_TOKEN_ID }}=${{ secrets.CI_RUNNER_API_TOKEN_SECRET }}"
+  NODE: ${{ secrets.PROXMOX_NODE }}
+  SSH_OPTS: "-o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o ConnectTimeout=10"
+  # Platform-sensitive packages whose runtime behaviour must be validated on a
+  # real kernel. Pure-Go packages (codec/crypto/logger/errs) behave identically
+  # everywhere and are covered by the Linux Bazel lane, so they are excluded.
+  PROC_PKGS_CORE: "./proc"
+  PROC_PKGS_SERVICE: "./proc/exec ./proc/cgroup ./proc/reaper ./proc/rlimit ./proc/sdlisten ./proc/sdnotify ./proc/signal ./logger/sink/file ./writer/rotfile"
+
+jobs:
+  e2e:
+    runs-on: [self-hosted, proxmox]
```

actionlint (1.7.12)

[error] label "proxmox" is unknown. available labels are "windows-latest", "windows-latest-8-cores", "windows-2025", "windows-2025-vs2026", "windows-2022", "windows-11-arm", "ubuntu-slim", "ubuntu-latest", "ubuntu-latest-4-cores", "ubuntu-latest-8-cores", "ubuntu-latest-16-cores", "ubuntu-24.04", "ubuntu-24.04-arm", "ubuntu-22.04", "ubuntu-22.04-arm", "macos-latest", "macos-latest-xlarge", "macos-latest-large", "macos-26-intel", "macos-26-xlarge", "macos-26-large", "macos-26", "macos-15-intel", "macos-15-xlarge", "macos-15-large", "macos-15", "macos-14-xlarge", "macos-14-large", "macos-14", "self-hosted", "x64", "arm", "arm64", "linux", "macos", "windows". if it is a custom label for self-hosted runner, set list of labels in actionlint.yaml config file

(runner-label)